spec/features/user/items/user_cant_see_add_to_cart_for_retired_item_in_index_spec.rb

```diff
@@ -3,7 +3,7 @@
 RSpec.describe 'As a visitor' do
 	describe 'when visiting an items show page' do
```

Comment:
create(:item) - get rid of URLs & any unnecessary attributes
category association is done within item factory